package.json

```diff
@@ -5,26 +5,29 @@
   "main": "index.js",
   "dependencies": {},
   "devDependencies": {
-    "bootstrap": "^3.4.1",
+    "@fortawesome/fontawesome-free": "^5.12.0",
+    "bootstrap": "^4.4.1",
     "bootstrap-datetime-picker": "^2.4.4",
     "eslint": "^6.7.2",
     "extend": ">=3.0.2",
     "gulp": "^4.0.2",
     "gulp-concat": "^2.6.1",
-    "gulp-less": "^4.0.1",
+    "gulp-sass": "^4.0.2",
     "gulp-sourcemaps": "^2.6.5",
     "inputmask": "^4.0.9",
     "jquery": "^3.4.1",
     "jquery-mousewheel": "^3.1.13",
     "jquery-sortable": "^0.9.13",
     "less": "^3.10.3",
+    "popper.js": "^1.15.0",
     "pump": "^3.0.0",
     "select2": "4.0.12",
     "select2-bootstrap-theme": "0.1.0-beta.10",
+    "tempusdominus-bootstrap-4": "^5.1.2",
```

Comment:
This library doesn't look maintained - there are several bugs which have not been addressed for months. We've stopped using it on our projects, and switched to using the native browser date input when available. I think this provides a better user experience, especially on mobile, and would suggest that Oscar defaults to this - projects can then add their own custom datepickers if they want to.

Reply:
Ok. Will remove it.

Reply:
I found that not all modern browsers support native input types:
* `time` - Safari
* `date` - Safari
* `datetime-local` - FF, Safari

Probably `date` and `time` is not a problem, because we can use `inputmask` for these fields (and they will work in Safari as well) 🤔
But `datetime-local` uses a format like this `2017-06-01T08:30` (I suppose we will need to add this format somewhere in settings globally) and `inputmask` for `datetime` is not "user-friendly" without the date picker (because of hours, minutes and seconds).

I agreed that it is better to use as fewer dependencies as possible, but I am not sure how we can completely remove the date picker.

How do you solve this in your projects? Should we use native input types with `inputmask` for each such field? Thanks.